The Telegraph.js

```diff
@@ -154,88 +113,163 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "newspaperArticle",
+				"title": "China: Google Earth spots huge, unidentified structures in Gobi desert",
 				"creators": [
 					{
 						"firstName": "Malcolm",
 						"lastName": "Moore",
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [
-					"China",
-					"Asia",
-					"World News",
-					"News"
-				],
-				"seeAlso": [],
+				"date": "2011-11-14",
+				"abstractNote": "Vast, unidentified, structures have been spotted by satellites in the barren Gobi desert, raising questions about what China might be building in a region it uses for its military, space and nuclear programmes.",
+				"archiveLocation": "8888909",
```

Comment:
This is the article id and was previously saved in the `callNumber`. Should we drop this information completely? Sometimes such internal database ids can be useful identifiers, but here I don't see any reuse of this number (and it is already part of the url). What do you suggest?